citelang/main/contrib.py

```diff
@@ -518,12 +518,22 @@ def get_commit_range(self, shallow=False):
         """
         Given a start and end, parse and return the commits from git
         """
+        # Single commit, honor previous behavior
+        if self.start == self.end:
```

Comment:
This is going to trigger when they are both == None, so you want to do
```suggestion
        if self.start == self.end and self.start:
```

Reply:
And we don't want either to be be None (it will print "None" as a string) so in the init you want to do:
```
class ContributionParser(GitParser):
    def __init__(self, root=None, start=None, end=None, outdir=None, paths=None):
        self.start = start or ""
        self.end = end or ""
```